schema.graphql

```diff
@@ -5831,6 +5832,7 @@ type Query {
   menuItems(channel: String, sortBy: MenuItemSortingInput, filter: MenuItemFilterInput, before: String, after: String, first: Int, last: Int): MenuItemCountableConnection
   giftCard(id: ID!): GiftCard
   giftCards(sortBy: GiftCardSortingInput, filter: GiftCardFilterInput, before: String, after: String, first: Int, last: Int): GiftCardCountableConnection
+  giftCardCurrencies: [String!]!
```

Comment:
Why?

Reply:
I thought it's latest schema

Reply:
Yes, but why copy it here when it's not connected to stock reserve? it might cause conflicts when working on the gift cards, so I'd just remove it from there

Reply:
sure, I'll fix it